games/minimal/mods/experimental/init.lua

```diff
@@ -703,8 +703,19 @@ minetest.register_chatcommand("test1", {
 				"list[current_player;craft;8,0;3,3;]"..
 				"list[current_player;craftpreview;12,1;1,1;]"..
 				"list[detached:test_inventory;main;0,0;4,6;0]"..
-				"button[0.5,7;2,1;button1;Button 1]"..
-				"button_exit[2.5,7;2,1;button2;Exit Button]"
+				"image_button[0.5,6;2,1;experimental_button_up.png;"..
```

Comment:
you should modify   `minimal/mods/tests/formspec.lua`

Reply:
Where are the style properties documented right now? I did a quick scan, but I seem to have missed them.

Reply:
lua_api.txt `Styling Formspecs`

Reply:
Both issues should be fixed now